test/util/math_sqrt.cpp

```diff
@@ -35,8 +35,9 @@
 
 namespace bg = boost::geometry;
 
+using namespace boost::unit_test;
```

Comment:
There are also other instances in other places, about the same issue; I have not commented on them.